### ISSUE TITLE
Requests for site use

### ISSUE THREAD

**Issue by mhasself** [simonsobs/soconda#22]
Thanks for this -- it is working great for me now and I am setting it up for general users.  Two requests for next time I build ... :

- Please update so3g to 0.1.12 (hash below)
- the jupyter kernel text is, e.g. "Simons Observatory 0.0.5.dev160".  In the icon view of kernels, this gets truncated to two lines, just "Simons Observatory".  Shorter form for display_name,  such as "soconda 0.0.5.dev160" would be less confusing.

```
{% set version = "0.1.12" %}
{% set sha256 = "d5f2a2af8948df3646f84abec669a771e2005a4e140afc1599529efcdc009364" %}
```

**Comment by tskisner** [comment]
Note that the first point here is addressed in #27.  Forgot to deal with second point, but will do that soon.

**Comment by tskisner** [comment]
Closed by #29
